rpcs3/Input/ds4_pad_handler.cpp

```diff
@@ -180,6 +181,18 @@ void ds4_pad_handler::init_config(pad_config* cfg, const std::string& name)
 	cfg->from_default();
 }
 
+u32 ds4_pad_handler::GetBatteryLevel(const std::string& padId)
+{
+	std::shared_ptr<DS4Device> device = GetDS4Device(padId);
+	if (device == nullptr || device->hidDevice == nullptr)
+	{
+		return 0;
+	}
+	GetRawData(device);
+	u32 battery_level = device->batteryLevel*10; // Return battery level in % instead of raw data DS4 reports to allow a unified way to display battery level for different controllers
```

Comment:
```suggestion

	// Return battery level in % instead of raw data DS4 reports to allow a unified way to display battery level for different controllers
	const u32 battery_level = device->batteryLevel * 10;
```